elephant/statistics.py

```diff
@@ -524,3 +525,101 @@ def instantaneous_rate(spiketrain, sampling_period, form, sigma, m_idx=None,
                                  units=pq.Hz, t_start=t_start)
 
     return rate
+
+
+def psth(spiketrains, binsize, t_start=None, t_stop=None, output='counts',
+         clip=False):
+    """
+    Peristimulus Time Histogram (PSTH) of a list of :attr:`neo.SpikeTrain`
```

Comment:
I think the issue here is that 'PSTH' is a specific use case for this function.  In many instances users (at least we here in Juelich) may not have a stimulus, but would still want to compute the time histogram over a set of SpikeTrain objects.  

For example, when there are, say, a hundred of so parallel recordings from an elecrode array and we wish to compute a so-called 'Population Time Histogram'.  We can use the 'psth' function for this, but then the name is not appropriate.  

My feeling is that the function should be made to be generic.

Therefore, I would like the function to be called time_histogram and to take t_start and t_stop as parameters, to allow the result to be computed only over simultaneous subsections of each input.